nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -38,12 +38,23 @@ def __init__(self):
         self.phases = queue.Queue()
 
     def go_to_patient(self):
-        # Extract room number and bed number from goal (bell)
-        status = self.navigate_task.execute(self.room, "bed")
+        rospy.loginfo("Nightingale Mission Planner going to patient")
+        # Assume door is open
+        # TODO: first go to doorside then bedside when door opening added
+        status = self.navigate_task.execute(self.room, "bedside")
         if status == Task.ERROR:
             raise NotImplementedError()
         self.phases.put(self.triage_patient)
 
+    def go_home(self):
+        rospy.loginfo("Nightingale Mission Planner going home")
+        # Assume door is open
+        # TODO: first go to doorside then bedside when door opening added
+        status = self.navigate_task.execute("home", "")
```

Comment:
The room pose lookup service requires a name for both of the room and sublocation. The sublocation for "home" is "idle"

Reply:
ill address this once i add the lookup funcationality to room runner